ACMILabs/spotlights
# Add/1481 collect

To test:
1. Run the server by following the readme
2. Open http://localhost:8081/
3. Run `curl -d '{}' -H "Content-Type: application/json" -XPOST http://localhost:8081/api/taps/` and see the "Collect ↓" change to "Collected ✓", and change back.
